keps/104-StartupPolicy/kep.yaml

```diff
@@ -0,0 +1,29 @@
+title: Startup Policy
+kep-number: 104
+authors:
+  - "@kannon92"
+status: provisional
+creation-date: 2023-08-01
+reviewers:
+  - "@daniel"
+  - "@ahg"
+approvers:
```

Comment:
Am I reviewer here or just an accessory reviewer? :laughing:

Reply:
Sorry, this is the first KEP so I did this very quick and a bit sloppy.  I'll add you as a reviewer!